great-idea/css/index.css

```diff
@@ -63,4 +58,177 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 876px;
+	margin: 0 auto;
+}
+
+.navigation {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	width: 876px;
+	margin-top: 25px;
+}
+
+.navigation a {
+	text-decoration: none;
+	color: grey;
+}
+
+header {
+	height: 450px;
+	text-align: center;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	margin-top: 25px;
+}
+
+.h1-div {
```

Comment:
I would like to see more use of specificity in your projects. So far this displays you grabbing elements directly through classes and other tags. What if you have more than one button that has a different style or want to reuse a classname but not have it be the same as another?